DEVELOPER_GUIDE.md

```diff
@@ -255,31 +224,34 @@ Use the command below to set up a Sink connector to a Authenticated Streaming Co
           "aep.flush.interval.seconds": 1,
           "aep.flush.bytes.kb": 4,
           "aep.connection.auth.enabled": true,
-          "aep.connection.auth.token.type": "jwt_token",
+          "aep.connection.auth.token.type": "oauth2_access_token",
           "aep.connection.auth.client.id": "<client_id>",
-          "aep.connection.auth.imsOrg": "<organization-id>",
-          "aep.connection.auth.accountKey": "<technical-account-id>",
-          "aep.connection.auth.filePath": "<path-to-private-pkcs8.key>",
+          "aep.connection.auth.client.secret": "<client_secret>"
           "aep.connection.auth.endpoint": "<ims-url>",
           "aep.connection.endpoint.headers": "<optional-header-that-needs-to-be-passed-to-AEP>"
-          "aep.connection.auth.client.secret": "<client_secret>"
         }
       }' http://localhost:8083/connectors
       ```
-      
-     Note - `aep.connection.endpoint.headers` format should be JSON-encoded. 
-     Example: To send below 2 HTTP headers - 
+
+     Note - `aep.connection.endpoint.headers` format should be JSON-encoded.
+     Example: To send below 2 HTTP headers -
      1. key: `x-adobe-flow-id`, value: `341fd4f0-cdec-4912-1ab6-fb54aeb41286`
      2. key: `x-adobe-dataset-id`, value: `3096fbfd5978431948af3ba3`
-   
+
+     `aep.connection.auth.endpoint` value for prod
```

Comment:
Could you please move aep.connection.auth.endpoint outside aep.connection.endpoint.headers block? We can add bullet to clearly differentiate between `aep.connection.endpoint.headers` and `aep.connection.auth.endpoint`